tests/schema/tests.py

```diff
@@ -136,6 +136,22 @@ def check_added_field_default(self, schema_editor, model, field, field_name, exp
                 database_default = cast_function(database_default)
             self.assertEqual(database_default, expected_default)
 
+    def count_constraints(self, table, column, to):
+        with connection.cursor() as cursor:
+            constraints = connection.introspection.get_constraints(cursor, table)
+        fks = 0
+        uniques = 0
+        indexes = 0
+        for c in constraints.values():
+            if c['columns'] == [column]:
+                if c['foreign_key'] == to:
+                    fks += 1
+                if c['unique']:
+                    uniques += 1
+                elif c['index']:
+                    indexes += 1
+        return fks, uniques, indexes
```

Comment:
It might improve readability to return a dictionary, `{'fks': x, 'uniques': y, 'indexes': z}` rather than having to remember the order.